app/src/main/cpp/Quad.cpp

```diff
@@ -236,6 +259,18 @@ Quad::SetBackgroundColor(const vrb::Color& aColor) {
   if (m.backgroundColor == aColor || (aColor.Alpha() == 0.0f && m.backgroundColor.Alpha() == 0.0f)) {
     return;
   }
+  if (m.layer) {
+    /*
+     * We disable the background when using layers because there is a difficult to fix glitch.
+     * We can create 2 nodes to simulate the background color on the sides of the ScaleToFit mode.
+     * It works while resizing, but at the end, when the actual GV surface is really changed
+     * there is some SurfaceTexture update delay that causes a glitch. Background color becomes
+     * transparent or the background color is shown on top of the GV quad surface.
+     * This glitch doesn't happen with normal scenegraph code because the background color is shown
+     * on the back of the Quad surface. But with Layers we can't do that.
+     */
+    return;
```

Comment:
I'm not sure I understand the issue but it sounds like we were drawing a blank quad while resizing. I'm surprised this didn't cover the layer since the content layer is drawn first? Feel free to file a follow up if there is still an out standing issue.